### ISSUE TITLE
In 'Currency' mode 'Update rates' is defined as a link instead of button

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#28]
Test Environment:
Calculator : 10.1805.1201.0
OS Build : 17685.1000.180531-1431

Repro Steps:

1. Start the narrator in scan mode
2. Launch calculator
3. Home screen will be displayed.
4. Navigate to menu bar and select 'Currency' Mode
5. Navigate to 'Update rates' link
6. Verify if narrator announces 'Updates rate' as link

Actual Result :
In 'Currency' mode 'Update rates' is defined as link

Expected Result:
In 'Currency' mode 'Update rates' should be button instead of link
